hi-ml-histopathology/src/histopathology/utils/download_utils.py

```diff
@@ -20,12 +20,12 @@ def download_file_if_necessary(run_id: str, remote_dir: Path, download_dir: Path
     """
     aml_workspace = get_workspace()
     os.chdir(fixed_paths.repository_root_directory())
-    local_path = download_dir / run_id.split(":")[1] / "outputs" / filename
+    local_path = download_dir / run_id.split(":")[1] / "outputs/test" / filename
```

Comment:
Why is it necessary to hardcode this "test/" subdirectory here? I'd avoid this, if possible, as I believe this is meant as a general-purpose function. If the desired file is now generated in a different directory, I suggest changing the caller to pass `filename="test/my_file"`.